apps/element-storybook/.storybook/preview.tsx

```diff
@@ -1,34 +1,32 @@
-import React from 'react';
-import { ThemeProvider } from '@availity/theme-provider';
 import { themes } from '@storybook/theming';
+import { ThemeProvider } from '@availity/theme-provider';
 
 export const parameters = {
   docs: {
+    controls: {
+      sort: 'requiredFirst',
```

Comment:
The only reason I've been hesitant to do this is because it also alphabetizes it, so some of the least used props will be up top. and with the amount of props and not the most intuitive naming might make the docs less legible for everyone coming over from bs4. 
the only real alternative is to  have default controls for required & most used so they're at the top which isn't that great either.

Reply:
I had similar reservations, but I think it's better than an arbitrary order